modules/percolator/src/test/resources/rest-api-spec/test/percolate/15_new.yaml

```diff
@@ -1,46 +0,0 @@
----
-"Basic percolation tests":
-
-  - do:
```

Comment:
Agreed, I added a basic test for both the search and msearch apis using a `percolate` query.